java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3664,4 +3664,30 @@ public void bitcount(BaseClient client) {
             assertTrue(executionException.getCause() instanceof RequestException);
         }
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void getbit(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        String missingKey = UUID.randomUUID().toString();
+        ;
+        String value = "foobar";
+
```

Comment:
I think you should also test GETBIT with SETBIT

Reply:
@GumpacG you can use client.customCommand(new String[] {"SETBIT", ...}).get() to test this for now.  When we merge, then you can update to the actual command.

Reply:
Oh I thought it was already merged, but yes custom command is good